downstream/modules/platform/proc-configuring-the-metrics-utility.adoc

```diff
@@ -3,23 +3,22 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="configuring-the-metrics-utility_{context}"]
-= configuring-the-metrics-utility
+= Configuring metrics-utility
```

Comment:
Did you want this heading to be Configuring the metrics-utility?

Reply:
I've been using the tool name without the definite article. I think Kyla must've created this module. I'll change it if you think it'll cause issues, otherwise I'd like to keep it consistent with the rest of the language in the in the assembly.